src/backend/wine/manager/utils.ts

```diff
@@ -32,12 +32,11 @@ async function updateWineVersionInfos(
   logInfo('Updating wine versions info', LogPrefix.WineDownloader)
   if (fetch) {
     logInfo('Fetching upstream information...', LogPrefix.WineDownloader)
+    const repositorys = isMac
```

Comment:
Did this mistake aswell :D
I thought i fixed that in my PR.

```suggestion
    const repositories = isMac
```

Reply:
I wanted to fix but now I want to trademark it. 😂 
Lets fix it another time.